xbmc/dialogs/GUIDialogBusyNoCancel.h

```diff
@@ -19,7 +19,7 @@ class CGUIDialogBusyNoCancel: public CGUIDialog
   void Render() override;
 
 protected:
-  void Open_Internal(const std::string &param = "") override;
+  void Open_Internal(bool bProcessRenderLoop, const std::string &param = "") override;
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types